includes/class-scd-ext-access-control.php

```diff
@@ -218,7 +218,12 @@ public function get_lesson_drip_date( $lesson_id , $user_id = '' ) {
 				return $drip_date;
 			}
 
+		if ( ! ctype_digit( $lesson_set_date ) ) {
```

Comment:
Extract into a public static method in `Scd_Ext_Utils` : (e.g. `$drip_date = Scd_Ext_Utils::date_from_datestring_or_timestamp( $lesson_id );`)